Iterable/iterable-android-sdk
# [MOB-9315] Written automated unit test cases for different field types and comparator types

##  Jira Ticket(s) if any

* [MOB-9315](https://iterable.atlassian.net/browse/MOB-9315)

##  Description

> Write automated unit tests for different field types and comparator types

##  Test Steps

> Video attached to the ticket, See a JIRA ticket for the video.


[MOB-9314]: https://iterable.atlassian.net/browse/MOB-9314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[MOB-9315]: https://iterable.atlassian.net/browse/MOB-9315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
